### PR TITLE
[ERE-1945] Embeddable registration form

### PR DESCRIPTION
**Pull request by ppettitau**
Persist language preference

### CODE REVIEW

File: rdrf/rdrf/views/registration_rdrf.py [hunk 1]
```diff
@@ -111,19 +111,36 @@ def dispatch(self, request, *args, **kwargs):
...
+            translation.activate(self.language_code or settings.LANGUAGE_CODE)
 
         return super().dispatch(request, *args, **kwargs)
 
+    def render_to_response(self, context, **response_kwargs):
+        response = super().render_to_response(context, **response_kwargs)
+
+        # Persist language preference
+        if self.activate_language:
+            response.set_cookie(settings.LANGUAGE_COOKIE_NAME, self.language_code)
```
Comment (ScriptSmith) [inline rdrf/rdrf/views/registration_rdrf.py]: Sorry thought I submitted this but it was still pending!

Can this be done in dispatch? Something like this:
```
request.session[settings.LANGUAGE_COOKIE_NAME] = language
```

**Review by ScriptSmith — CHANGES_REQUESTED** [review]

File: rdrf/rdrf/views/registration_rdrf.py (hunk 1 above)
Comment (ppettitau) [reply rdrf/rdrf/views/registration_rdrf.py]: I can try.

Comment (ppettitau) [reply rdrf/rdrf/views/registration_rdrf.py]: Doesn't work for me

There used to be a session variable for setting the language, but it's deprecated in Django 3, removed in Django 4. 
https://github.com/django/django/commit/a8e2a9bac6e548d6ab2e13af6171d2fdd3b8055b

Also, the django docs have a list of series of locations to determine the user's language preference, and the session is not in any of them
https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#how-django-discovers-language-preference

Comment (ScriptSmith) [reply rdrf/rdrf/views/registration_rdrf.py]: :+1: I think overriding `render_to_response` is unnecessary, so `response.set_cookie(...)` can be done in dispatch:

```python
    def dispatch(self, request, *args, **kwargs):
        response = super().dispatch(request, *args, **kwargs)

        if language := request.GET.get('language'):
            translation.activate(language)
            response.set_cookie(settings.LANGUAGE_COOKIE_NAME, language)

        return response
```

**Review by ScriptSmith — APPROVED** [review]

**Pull request merged**